### ISSUE TITLE
Re-add the use of connectors on `create-fuels` template

### ISSUE THREAD

**Issue by Torres-ssf** [FuelLabs/fuels-ts#2186]
During https://github.com/FuelLabs/fuels-ts/pull/2021 we had to stop using [connectors](https://github.com/FuelLabs/fuel-connectors) within the `fuels-create` [template](https://github.com/FuelLabs/fuels-ts/pull/2021/commits/a1fcca7bb365f848f6ac80c789a28841c0c0a56b) 

This was done to avoid `fuel-core` related errors when using bootstrapped projects with `fuels-create`. 

The error happens because the connectors repository is not using the most recent version from `fuels`.

**Comment by Torres-ssf** [comment]
Blocked by https://github.com/FuelLabs/fuel-connectors/pull/57

**Comment by arboleya** [comment]
Duplicate of https://github.com/FuelLabs/fuels-ts/issues/2171
